assets/js/atomic/blocks/product-elements/product-details/style.scss

```diff
@@ -0,0 +1,46 @@
+.wp-block-woocommerce-product-details {
```

Comment:
```suggestion
.wc-block-woocommerce-product-details {
```

Reply:
see https://github.com/woocommerce/woocommerce-blocks/pull/8225#discussion_r1106985254 for context.